recipes/cantera/.gitattributes

```diff
@@ -0,0 +1 @@
+*.bat text eol=lf 
```

Comment:
I don't develop on Windows so I don't really now how this affects this recipe. Does it make sense to add it to the global settings for this repo?

Reply:
Without this, git (on Linux) warned me that it had changed the line endings, which I'm not sure `cmd` would  view favorably.

Reply:
Yeah that’s expected. Linux would have used `lf`, which we replace with `crlf`. So git is just mentioning it did that which is fine

Reply:
OIC. In that case, I'll delete this.

Reply:
We should update staged-recipes `.gitattributes` file to match that of feedstocks.